js-core/homeworks/homework-3/src/main.js

```diff
@@ -0,0 +1,161 @@
+/*
+Напишите функцию которая будет возвращать объект
+с свойством name, а значением будет первый аргумент функции
+*/
+
+function returnObject(name) {
+    const someObject = name
```

Comment:
someObject is not really object :)

You set to variable someObject value equal to passed prop 'name'.

So someObject is just a string.

You have to create new object

Reply:
yes! 
but please make it multi-line